### PR TITLE
Making a change

### PR DESCRIPTION
**Pull request by EmilioGarcia12**

### CODE REVIEW

File: src/main/webapp/index.jsp [hunk 1]
```diff
@@ -21,7 +21,7 @@
     <div
       style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
 
-      <h1>CodeU Chat App</h1>
+      <h1>Lol this is hard</h1>
```
Comment (rhbrady) [inline src/main/webapp/index.jsp]: You should spell out Lol

Comment (wgenemorgan) [inline src/main/webapp/index.jsp]: This might be a bit too intimidating.

Comment (wgenemorgan) [reply src/main/webapp/index.jsp]: I suggest a friendlier message.

**Review by wgenemorgan — APPROVED** [review]
LGTM

File: src/main/webapp/index.jsp (hunk 1 above)
Comment (EmilioGarcia12) [reply src/main/webapp/index.jsp]: I fixed it. done

Comment (rhbrady) [reply src/main/webapp/index.jsp]: Cool

**Pull request merged**